models/git/commit_status.go

```diff
@@ -322,7 +323,7 @@ type SignCommitWithStatuses struct {
 }
 
 // ParseCommitsWithStatus checks commits latest statuses and calculates its worst status state
-func ParseCommitsWithStatus(ctx context.Context, oldCommits []*asymkey_model.SignCommit, repo *repo_model.Repository) []*SignCommitWithStatuses {
+func ParseCommitsWithStatus(ctx context.Context, oldCommits []*asymkey_model.SignCommit, repo *repo_model.Repository, lang translation.Locale) []*SignCommitWithStatuses {
```

Comment:
It doesn't look good, I guess it's better to decouple the git package from translation package?